apps/web/src/components/Solana/ConnectWalletModal.tsx

```diff
@@ -0,0 +1,169 @@
+"use client";
+
+import { Box, Button, Text } from "@dex-web/ui";
+import type { WalletName } from "@solana/wallet-adapter-base";
```

Comment:
We should discuss usage of the Solana SDK, as it would be good to have a plan before we commit to any patterns! I.e. usage server side. Have spoken to Vitor about this a little, worth having a chat about together.